File: 01_Language/01_Functions/js/ctype_digit.js
function ctype_digit(text) {
    if (text === null || text === undefined || text === '') return false
    for (let i in text) {
        let codePoint = text.charCodeAt(i)
        if (codePoint >= 48 && codePoint <= 57) {
            continue
        }
        return false
    }
    return true
}

File: 01_Language/01_Functions/go/src/functions/ctype_digit.go
package functions

func CtypeDigit(s string) bool {
	if len(s) == 0 {
		return false
	}
	for i := 0; i < len(s); i++ {
		codePoint := int(s[i])
		if codePoint >= 48 && codePoint <= 57 {
			continue
		}
		return false
	}
	return true
}
